### PR TITLE
Darwinia going live

### PR DESCRIPTION
**Pull request by isunaslabs**
-Added Darwinia 2.0 configurations

### CODE REVIEW

File: packages/app-config/src/chains/darwinia.ts [hunk 1]
```diff
@@ -27,9 +27,22 @@ export const darwinia: ChainConfig = {
   },
   chainId: 46,
   substrate: {
-    graphQlURL: "https://subql.darwinia.network/subql-apps-darwinia/",
-    wssURL: "wss://rpc.darwinia.network",
-    httpsURL: "https://rpc.darwinia.network",
+    graphQlURL: "https://api.subquery.network/sq/isunaslabs/darwinia2",
```
Comment (hackfisher) [inline packages/app-config/src/chains/darwinia.ts]: @fewensa please review this.

File: packages/app-config/src/chains/darwinia.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ export const darwinia: ChainConfig = {
   name: "Darwinia",
   displayName: "Darwinia",
   explorerURLs: ["https://darwinia.subscan.io/"],
-  httpsURLs: ["https://rpc.darwinia.network"],
+  httpsURLs: ["https://cors.zimjs.com/http://g1.dev.darwinia.network:10000"],
```
Comment (hackfisher) [inline packages/app-config/src/chains/darwinia.ts]: @isunaslabs Please fix this.

File: packages/app-config/src/chains/darwinia.ts [hunk 3]
```diff
@@ -27,9 +27,22 @@ export const darwinia: ChainConfig = {
   },
   chainId: 46,
   substrate: {
-    graphQlURL: "https://subql.darwinia.network/subql-apps-darwinia/",
-    wssURL: "wss://rpc.darwinia.network",
-    httpsURL: "https://rpc.darwinia.network",
+    graphQlURL: "https://api.subquery.network/sq/isunaslabs/darwinia2",
+    wssURL: "ws://g1.dev.darwinia.network:20000",
+    httpsURL: "https://cors.kahub.in/http://g1.dev.darwinia.network:10000",
```
Comment (hackfisher) [inline packages/app-config/src/chains/darwinia.ts]: @isunaslabs Fix this.

**Review by hackfisher — CHANGES_REQUESTED** [review]

File: packages/app-config/src/chains/darwinia.ts (hunk 3 above)
Comment (isunaslabs) [reply packages/app-config/src/chains/darwinia.ts]: fixed

File: packages/app-config/src/chains/darwinia.ts (hunk 2 above)
Comment (isunaslabs) [reply packages/app-config/src/chains/darwinia.ts]: fixed

File: packages/app-config/src/chains/darwinia.ts (hunk 1 above)
Comment (isunaslabs) [reply packages/app-config/src/chains/darwinia.ts]: fixed

**Comment by 1022ranji** [comment]
使用staking Ui 选择 collator 是出现“unknown error”
address: 0x32C139e80Dee4125f500b51A0365A8522Cf352D0
https://user-images.githubusercontent.com/80877629/233995518-238b1949-f114-45fa-bf02-9a70822dce27.mov

**Review by hackfisher — APPROVED** [review]

File: yarn.lock [hunk 4]
```diff
@@ -3418,10 +3418,10 @@ darwinia-js-sdk@^1.2.18:
     "@polkadot/types" "^9.9.4"
     ethers "^5.7.2"
 
-darwinia-js-sdk@^2.0.3:
-  version "2.0.3"
-  resolved "https://registry.yarnpkg.com/darwinia-js-sdk/-/darwinia-js-sdk-2.0.3.tgz#6ba16a8eb3867cbb0c04660e708cbb14b67c7fe2"
-  integrity sha512-OToeoAJvBJK/qid1PFg+9/nxydja3Ld3Wq1zaMaFi3exI2AO3xHdCHRslNm2tcBs4iG07UO4Phv0ZxDO1nzUSg==
+darwinia-js-sdk@^2.0.4:
```
Comment (hackfisher) [inline yarn.lock:3421]: Do yarn.lock have high priority over package.json

**Pull request merged**